basestar-bom/pom.xml

```diff
@@ -24,11 +24,11 @@
         <unirest.version>3.4.01</unirest.version>
         <avro.version>1.11.0</avro.version>
         <scala.version>2.12.15</scala.version>
-        <spark.version>3.0.1</spark.version>
-        <hadoop.version>2.8.5</hadoop.version>
+        <spark.version>3.2.1</spark.version>
```

Comment:
should we drop the custom profile config at the bottom of the file then?
```
    <profiles>
        <profile>
            <id>jdk9plus</id>
            <activation>
                <jdk>[9,)</jdk>
            </activation>
            <properties>
                <spark.version>3.2.1</spark.version>
            </properties>
        </profile>
    </profiles>
```

Reply:
Done - so that's how it's getting pulled to 3.2.1 in stage-depot-platform